src/window.cpp

```diff
@@ -3061,38 +3061,44 @@ void InputLoop()
 	HandleAutoscroll();
 }
 
+static const uint TICK_15_INTERVAL = 15 * MILLISECONDS_PER_TICK;
```

Comment:
If I understand you correctly, you want to say here "60 Hz"? Possibly more sane to define it as such? (as it is the update frequency; we are all used to it being in Hz I guess)

(so basically just `1000 * MILLISECONDS_PER_TICK / 60` I guess?)

Reply:
I wanted to clarify what the constant is about since the code using this had originally 15 ticks.

Reply:
The code using this value wants to achieve a delay of 450ms. Originally it just incremented a value since the function was called each tick and it would run the condition once the variable hit 15. I suppose I could use better names since its representing real time values.